adding-a-new-module/README.md

```diff
@@ -0,0 +1,128 @@
+## Template 
+This **project** includes all common dependencies and settings for kaluga module.
+
+#Steps for adding new module
+1. CopyPaste template module, select name for your module (we recommend to use only lowercase characters).
+2. `<your module>src/androidLibMain/AndroidManifest.xml` change `com.splendo.kaluga.template` to your module package name.	
+3. `<your module>src/androidLibAndroidTests/AndroidManifest.xml` change `com.splendo.kaluga.template` to your module package name.
+4. `<your module>src/androidLibAndroidTests/kotlin/TestActivity.kt` change package name from `com.splendo.kaluga.template` to `com.splendo.kaluga.your module`.
+5. Include your module to kaluga project
+	go to `kaluga/settings.gradle/kts`
+	And add in the end of the file:
+	`include(":<your module>")`.
+6. Add Unit tests
+    * [Common Unit Tests](#commonTests)
+    * [iOS Unit Tests](#iosTests)
+    * [Android Unit Tests](#androidLibTests)
+7. [Add Android Instrumented Unit Tests](#instrumentedUnitTests)
+8. Add your source code
+    * <a name="commonMain-sources"></a>**commonMain** - non-platform-specific implementation and `expect` declaration.
+    * <a name="iosMain-sources"></a>**iosMain** - iOS specific implementation.
+    * <a name="androidLibMain-sources"></a>**androidLibMain** - Android specific implementation.
+    * <a name="jvmMain-sources"></a>**jvmMain** - jvm specific implementation.
+    * <a name="jsMain-sources"></a>**jvmMain** - js specific implementation.
+8. Remove template README files from your module when you don't need them anymore.
+9. Don't forget to update copyright if files were imported from another project. To do that use **Code > Update Copyright...**.
+10. Add documentation to your code if needed.
+11. Add an example of usage to the example project.
+
+---
+### <a name="commonTests"></a> Common Unit Tests 
```

Comment:
FYI anchors are generated by GitHub based on the title of a heading automatically

Reply:
(no problem to leave them in however, these are technically more stable)